src/misceval.js

```diff
@@ -1189,6 +1189,14 @@ Sk.misceval.applyOrSuspend = function (func, kwdict, varargseq, kws, args) {
         //append kw args to args, filling in the default value where none is provided.
         return func.apply(null, args);
     } else {
+        fcall = func.__get__;
+        if (fcall !== undefined) {
+            fcall = Sk.misceval.callsimOrSuspend(func.__get__, func, func);
+            //args.unshift(func);
+            if (fcall.tp$call) {
```

Comment:
This will fail if `func.__get__` returns a suspension. What are you trying to do here?